testsuite/integration-arquillian/test-apps/spring-boot-adapter-app/pom.xml

```diff
@@ -192,7 +192,7 @@
             <dependency>
                 <groupId>org.springframework.boot</groupId>
                 <artifactId>spring-boot-starter-parent</artifactId>
-                <version>${spring-boot.version}</version>
+                <version>2.6.15</version>
```

Comment:
It's better to use a property instead of a hardcoded value. If you need Spring Boot 2.6.x version, use `spring-boot26.version` property from parent pom. There you can upgrade it to 2.6.15 if needed.

Reply:
I did this just for testing purposes, we need to remove this change completely